src/codegen/operations.rs

```diff
@@ -107,6 +109,76 @@ fn codegen_add<'c, 'r>(
     Ok((start_block, ok_block))
 }
 
+fn codegen_div<'c, 'r>(
+    op_ctx: &mut OperationCtx<'c>,
+    region: &'r Region<'c>,
+) -> Result<(BlockRef<'c, 'r>, BlockRef<'c, 'r>), CodegenError> {
+    let start_block = region.append_block(Block::new(&[]));
+    let context = &op_ctx.mlir_context;
+    let location = Location::unknown(context);
+
+    // Check there's enough elements in stack
+    let flag = check_stack_has_at_least(context, &start_block, 2)?;
+
+    // Create REVERT block
+    let revert_block = region.append_block(generate_revert_block(context)?);
```

Comment:
Just fresh out of the oven!
```suggestion
    let revert_block = region.append_block(op_ctx.revert_block);
```

Reply:
Just for posterity: this doesn't work since `append_block` takes ownership. `op_ctx.revert_block` is already registered in the region, and so you can just do:

```rust
let revert_block = op_ctx.revert_block;
```